src/components/themed/WalletListHeader.js

```diff
@@ -17,10 +18,14 @@ import { type Theme, type ThemeProps, cacheStyles, withTheme } from '../services
 import { EdgeText } from '../themed/EdgeText.js'
 import { PromoCard } from '../themed/PromoCard.js'
 import { WiredBalanceBox } from '../themed/WiredBalanceBox.js'
+import { EdgeTextFieldOutlined } from './EdgeTextField.js'
 
 type OwnProps = {
   sorting: boolean,
-  toggleSorting: (sorting: boolean) => void
+  searching: boolean,
+  toggleSorting: (sorting: boolean) => void,
```

Comment:
This function is not a toggle but a function to notify a change in sorting state. Change function name to onChangeSortingState